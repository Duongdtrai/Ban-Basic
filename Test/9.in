4
42
249 363 134 894 5 267 713 999 813 968 168 800 375 319 487 797 319 2 522 445 106 195 74 397 259 787 386 121 116 562 474 58 821 753 979 784 639 58 59 782 452 56 
35
486 269 777 442 799 207 586 411 907 836 187 116 804 500 375 361 869 115 252 869 750 894 47 769 37 415 915 858 952 473 400 875 697 867 39 
15
493 813 690 145 13 38 651 13 579 192 749 572 293 751 383 
2
584 395 
