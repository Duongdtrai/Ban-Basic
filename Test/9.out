34883853480
28307935104
254319728
0
